### ISSUE TITLE
lxml library import error on python 3.6 runtime

### ISSUE THREAD

**Issue by cowcow02** [Miserlou/Zappa#810]
## Context

I have been using Zappa for a few services and everything worked perfectly well! With the latest release of Python 3.6 support for AWS lambda, I was trying to adopt a python 3.6 library that is using the lxml package. When it is deployed to lambda, it returns `ImportError` when importing lxml.

 

## Expected Behavior

lxml package can be imported properly on python 3.6 runtime

## Actual Behavior

```
[1493227052155] File "/var/task/services/my_lib.py", line 7, in <module>
[1493227052155] import lxml.html
[1493227052155] ImportError: bad magic number in 'lxml': b'\x03\xf3\r\n'
[1493227052155] During handling of the above exception, another exception occurred:
[1493227052155] Traceback (most recent call last):
[1493227052155] File "/var/runtime/awslambda/bootstrap.py", line 466, in <module>
[1493227052155] main()
[1493227052155] File "/var/runtime/awslambda/bootstrap.py", line 462, in main
[1493227052155] handle_event_request(request_handler, invokeid, event_body, context_objs, invoked_function_arn)
[1493227052155] File "/var/runtime/awslambda/bootstrap.py", line 248, in handle_event_request
[1493227052214] result = to_json(result)
[1493227052214] File "/var/runtime/awslambda/bootstrap.py", line 224, in to_json
[1493227052214] return json.dumps(obj, default=decimal_serializer)
[1493227052214] File "/var/lang/lib/python3.6/json/__init__.py", line 238, in dumps
[1493227052214] **kw).encode(obj)
[1493227052214] File "/var/lang/lib/python3.6/json/encoder.py", line 199, in encode
[1493227052214] chunks = self.iterencode(o, _one_shot=True)
[1493227052214] File "/var/lang/lib/python3.6/json/encoder.py", line 257, in iterencode
[1493227052214] return _iterencode(o, 0)
[1493227052214] File "/var/runtime/awslambda/bootstrap.py", line 42, in decimal_serializer
[1493227052214] raise TypeError(repr(o) + " is not JSON serializable")
[1493227052214] TypeError: <FrameSummary file /var/task/handler.py, line 514 in lambda_handler> is not JSON serializable
[1493227052843] bad magic number in 'lxml': b'\x03\xf3\r\n'
```
## Possible Fix

I think it is due to the pre-complied lxml package in https://github.com/Miserlou/lambda-packages is not working well with the Python 3.6 runtime, just like https://github.com/Miserlou/Zappa/issues/800
## Steps to Reproduce

1. set lambda python runtime as 3.6
2. import lxml in the code 

I have also tried importing lxml on python 2.7 runtime, things working well without issue.

## Your Environment

* Zappa version used: 0.41.1
* Operating System and Python version: Mac OS Sierra, Python 3.6

**Comment by nikbora** [comment]
Should work on master as it will pull the binary from PyPi. That is assuming you are using a recent lxml version that has a PyPi manylinux version for 3.6 (e.g. [lxml-3.7.3-cp36-cp36m-manylinux1_x86_64.whl](https://pypi.python.org/pypi/lxml/3.7.3))

**Comment by Miserlou** [comment]
Another good reason to get https://github.com/Miserlou/lambda-packages/pull/45/files merged - hope to get to this soon, maybe today..

**Comment by nikbora** [comment]
If you need more motivation then here are the shenanigans I am having to perform on our CI server to force the downloading of manylinux wheels for numpy and psycopg2:

`echo "lambda_packages = {}" > ${ENV_PATH}/lib/python3.6/site-packages/lambda_packages/__init__.py`

😂

**Comment by cowcow02** [comment]
I can confirm the above issue is resolved with the 0.41.2 release (along with lambda-package 0.15.0). Thank you very much for the hard work!

**Comment by piyushmittal25** [comment]
I am facing the same problem with zappa version **0.51.0**
